static/templates/settings/data-exports-admin.handlebars

```diff
@@ -0,0 +1,18 @@
+<div id="data-exports" class="settings-section" data-name="data-exports-admin">
+    {{#if is_admin}}
+    <div class="alert" id="export_status" role="alert">
+        <span class="export_status_text"></span>
+    </div>
+    <form class="form-horizontal">
+        <p class="alert-word-settings-note">{{#tr this}}This tool does a <a href="https://zulipchat.com/help/export-your-organization">public export</a> of your organization's history.{{/tr}}</p>
```

Comment:
please use `target=_blank` on the /help/ link so it opens in a new tab, and a relative link.  See `static/templates/settings/linkifier-settings-admin.handlebars` for a good example of this sort of note widget.